source/composeK.js

```diff
@@ -39,5 +38,9 @@ export default function composeK() {
   }
   var init = Array.prototype.slice.call(arguments);
   var last = init.pop();
-  return compose(compose.apply(this, map(chain, init)), last);
+  if (init.length === 0) {
+    last;
+  } else {
```

Comment:
Assuming previous comment is a return, isn't the else here unneeded?

Reply:
Fair point. we could do something like this
```js
if (init.length === 0) {
    return last;
  }
  return compose(compose.apply(this, map(chain, init)), last);
```